cursorless-talon/src/cheatsheet/sections/scopes.py

```diff
@@ -4,7 +4,7 @@
 def get_scopes():
     return {
         **get_lists(
-            ["scope_type", "selection_type", "subtoken_scope_type"],
+            ["scope_type", "selection_type", "subtoken_scope_type", "head_tail"],
```

Comment:
Not sure I like this but thinking of a better description...

![image](https://user-images.githubusercontent.com/1354217/171285572-d4477d70-b676-4e86-8ed1-5b93d6a9b8c4.png)

Reply:
I think this will have broken with merger of #672.  We now have one list called something like `simple_modifier` that includes "head", "tail" and a few other things